app/models/vmdb_table.rb

```diff
@@ -6,6 +6,35 @@ class VmdbTable < ApplicationRecord
 
   has_one  :latest_hourly_metric,  -> { VmdbMetric.where(:capture_interval_name => 'hourly', :resource_type => 'VmdbTable', :timestamp => VmdbMetric.maximum(:timestamp)) }, :as => :resource, :class_name => 'VmdbMetric'
 
+  # index_name1,unique1,col11,col12|index_name2,unique2,col21,col22
+  virtual_attribute :all_indexes, :string, :arel => (lambda do |t|
+    t.grouping(
+      Arel::Nodes::SqlLiteral.new(
+        "SELECT
+           string_agg(
+           distinct ix.relname || ',' || indisunique || ',' ||
+          regexp_replace(pg_get_indexdef(indexrelid), '^[^\\)]*\\(([^\\)]*)\\).*$', '\\1'), '|')
```

Comment:
yowsers... where's my arel buddy @NickLaMuro... can you review this PR when you get a chance?

Reply:
only @jrafanie would ask me to review from a diff comment...

looking now...

Reply:
Also... this is basically straight SQL at this point, and hardly `AREL` (#pedanticNickIsPedantic)

Reply:
> only @jrafanie would ask me to review from a diff comment...

@NickLaMuro you know me too well.  I get in deep and need a lifeline to save me.... you're my phone a friend



> Also... this is basically straight SQL at this point, and hardly `AREL`

🤣